chandra_aca/aca_image.py

```diff
@@ -305,9 +298,8 @@ def centroid_fm(self, bgd=None, pix_zero_loc="center", norm_clip=None):
 
         return row, col, norm
 
-    @override__dir__
     def __dir__(self):
-        return list(self.meta)
+        return sorted(super().__dir__() + list(self.meta))
```

Comment:
it does not work with `list`, it has to be `set`

Reply:
It works for me. `super().__dir__()` returns a list.

Strictly speaking `__dir__()` is required to return a sequence (https://docs.python.org/3/reference/datamodel.html#customizing-module-attribute-access). But in this case the class inherits from `ndarray` and that does indeed a `list` for `__dir__()`.